src/main/java/Duke/Ui.java

```diff
@@ -0,0 +1,58 @@
+package main.java.Duke;
+
+import java.util.Scanner;
+
+public class Ui {
+    Scanner scanner;
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Reads the next input from the user using a Scanner.
+     * @return User input in a string.
+     */
+    public String readCommand() {
+        return scanner.nextLine();
+    }
+
+    /**
+     * Greets the user when the program is executed.
+     */
+    public void greetingMessage() {
+        System.out.println("Hello! I'm Duke. What can I do for you?");
+    }
+
+    /**
+     * Prompts the user for an input
+     */
+    public void prompt() {
+        System.out.println("Input:");
+    }
+
+    /**
+     * When the user input is invalid, prints the list of possible input for the user.
+     */
+    public String badInput() {
+        return (
+                "Please input:\n"
+                + "1)list - to access the list\n"
+                + "2)todo - to create a todo task\n"
+                + "3)deadline - to create a deadline\n"
+                + "4)event - to schedule an event\n"
+                + "5)done - to mark tasks as done\n"
+                + "6)delete - to delete tasks from the list"
```

Comment:
It is good that you considered creating a pleasant user experience by making it easier for users to enter correct commands!